.travis.yml

```diff
@@ -50,7 +50,7 @@ script:
       travis_terminate 1
       ;;
     esac'
-  - pip install -r requirements.txt
```

Comment:
This is not good, you should move `-e git+http://github.com/nbs-system/nxapi-ng/#egg=nxapi` into the requirements.txt file, shouldn't you?